FRA_WE_BACK/core/utils.py

```diff
@@ -0,0 +1,25 @@
+import jwt
+
+from django.http 	        import JsonResponse    
+
+from FRA_WE_BACK.settings     import SECRET_KEY                             
+from users.models             import User
+
+
+def LogInDecorator(func):
+    def wrapper(self, request, *args, **kwargs):
+        try:
+            access_token = request.headers.get('Authorization', None)          
+            payload      = jwt.decode(access_token, SECRET_KEY, algorithms='HS256')  
+            user   	     = User.objects.get(id=payload['id'])                 
+            request.user = user
```

Comment:
suggestion과 같이 한 번에 지정할 수 있을 것 같습니다.
```suggestion
            access_token = request.headers.get('Authorization', None)          
            payload      = jwt.decode(access_token, SECRET_KEY, algorithms='HS256')  
            request.user = User.objects.get(id=payload['id'])                 
```